package.json

```diff
@@ -78,8 +80,8 @@
 					"varsIgnorePattern": "^React$"
 				}
 			],
-			"@typescript-eslint/explicit-function-return-type": "off",
-			"@typescript-eslint/restrict-plus-operands": "off"
+			"@typescript-eslint/explicit-function-return-type": "on",
+			"@typescript-eslint/restrict-plus-operands": "on"
```

Comment:
Just remove these lines to the enable the rules.

Reply:
Done.